_articles/ca-ES/building-community.md

```diff
@@ -1,56 +1,56 @@
+Una comunitat acollidora és una inversió per al futur i la reputació del teu projecte. Si el teu projecte està tot just veient les seves primeres contribucions, comença per donar una experiència positiva als primers contribuïdors i facilita que continuint venint.
 
-### Make people feel welcome
+### Fes que la gent se senti benvinguda
 
-One way to think about your project's community is through what @MikeMcQuaid calls the [contributor funnel](https://speakerdeck.com/mikemcquaid/the-open-source-contributor-funnel):
+Una manera de pensar sobre la comunitat del teu projecte es a través del que @MikeMcQuaid anomena [l'embut de contribució](https://speakerdeck.com/mikemcquaid/the-open-source-contributor-funnel):
 
-![Contributor funnel](/assets/images/building-community/contributor_funnel_mikemcquaid.png)
+![Embut de contribució](/assets/images/building-community/contributor_funnel_mikemcquaid.png)
 
-As you build your community, consider how someone at the top of the funnel (a potential user) might theoretically make their way to the bottom (an active maintainer). Your goal is to reduce friction at each stage of the contributor experience. When people have easy wins, they will feel incentivized to do more.
+A mesura que construeixes la teva comunitat, considera com algú que està al capdamunt de l'embut (un usuari potencial) pot obrir-se camí fins al fons (un matenidor actiu). El teu objectiu és reduir la fricció a cada etapa de l'experiència del contribuidor. Quan les persones tenen petites victòries, se senten incentivades per fer més.
 
-Start with your documentation:
+Comença amb la teva documentació:
 
-* **Make it easy for someone to use your project.** [A friendly README](../starting-a-project/#writing-a-readme) and clear code examples will make it easier for anyone who lands on your project to get started.
-* **Clearly explain how to contribute**, using [your CONTRIBUTING file](../starting-a-project/#writing-your-contributing-guidelines) and keeping your issues up-to-date.
+* **Facilita que la gent utilitzi el teu projecte** [Un README amigable](../starting-a-project/#writing-a-readme) i examples de codi clars facilitaran que algú que topa amb el teu projecte començi a usar-lo.
+* **Explica de manera clara com contribuir**, utilitzant [el teu arxiu CONTRIBUTING](../starting-a-project/#writing-your-contributing-guidelines) i mantenint els "issues" al dia.
 
-[GitHub's 2017 Open Source Survey](http://opensourcesurvey.org/2017/) showed incomplete or confusing documentation is the biggest problem for open source users. Good documentation invites people to interact with your project. Eventually, someone will open an issue or pull request. Use these interactions as opportunities to move them down the funnel.
```

Comment:
You are missing the translation of this part:

"Good documentation invites people to interact with your project. Eventually, someone will open an issue or pull request. Use these interactions as opportunities to move them down the funnel."

Reply:
Ups, sorry I should have said this is still work in progress

Reply:
👍